### PR TITLE
Bump klog to v0.3.2

### LINKED ISSUE #53: Repeated trace in log file with error
/kind bug

**What steps did you take and what happened:**
https://play.golang.com/p/6l8KNG_iOsM

Configuring klog to write to a file and logtostderr=false, the log file is openned and wrote three times with the same content.

Removing the flush, writes three times the headers, but no content.

**What did you expect to happen:**
Only one open and write.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

### PR DESCRIPTION
**Pull request by yuwenma**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

 /kind cleanup

> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes kubernetes/klog#53 #77904

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by BenTheElder** [comment]
changes: https://github.com/kubernetes/klog/compare/v0.3.1...v0.3.2

**Comment by BenTheElder** [comment]
These failures look like actual problems with the tests:
```
INFO: From Testing //staging/src/k8s.io/apimachinery/pkg/util/runtime:go_default_test:
==================== Test output for //staging/src/k8s.io/apimachinery/pkg/util/runtime:go_default_test:
--- FAIL: TestHandleCrashLog (0.00s)
runtime_test.go:99: panic log should have 1 line of message, 1 line per goroutine and 2 lines per function call
FAIL
================================================================================
==================== Test output for //staging/src/k8s.io/apimachinery/pkg/util/runtime:go_default_test:
--- FAIL: TestHandleCrashLog (0.00s)
runtime_test.go:99: panic log should have 1 line of message, 1 line per goroutine and 2 lines per function call
FAIL
================================================================================
==================== Test output for //staging/src/k8s.io/apimachinery/pkg/util/runtime:go_default_test:
--- FAIL: TestHandleCrashLog (0.00s)
runtime_test.go:99: panic log should have 1 line of message, 1 line per goroutine and 2 lines per function call
FAIL
================================================================================
```
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/78465/pull-kubernetes-bazel-test/1133623934294429696

File: Godeps/LICENSES [hunk 1]
```diff
@@ -231,7 +231,7 @@ COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
 IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-= vendor/bitbucket.org/bertimus9/systemstat/LICENSE 6f7ba8122a9b3146266dafa39c5b8ee3
+= vendor/bitbucket.org/bertimus9/systemstat/LICENSE
```
Comment (BenTheElder) [inline Godeps/LICENSES]: thanks to @cblecker it's clear that these should not be removed. the changes in this file are wrong.

Comment (yuwenma) [reply Godeps/LICENSES]: I'm not sure about  quite sure about the standard process of updating a vendor package. For klog, I think these are the suggested cmds: 
>> hack/pin-dependency.sh k8s.io/klog <GIT-COMMIT>
>> hack/update-vendor.sh

I can manually fix the problematic file. But I think the hack scripts may need to be updated as well. Do you know which sig or owner should be aware of the generic deps updates.

Comment (yuwenma) [reply Godeps/LICENSES]: oh, never mind. I saw the prow test gives the suggested fixes.

**Comment by liggitt** [comment]
reverted in https://github.com/kubernetes/kubernetes/pull/78931 due to performance regressions

**Pull request merged**